### ISSUE TITLE
How to use minibatch_size_schedule in CNTK 2.2?

### ISSUE THREAD

**Issue by aschweig** [Microsoft/CNTK#2480]
My task seems to benefit from having small mini-batches early and larger ones later.  I'd like to coordinate this using a `training_session`.  In the sources (e.g., `training_session_test.py` and `MNIST_Complex_Training.py`), I see `minibatch_size_schedule` is used with `learning_rate_schedule` + `UnitType.sample`.  Presumably gradients shrink, and using a larger minibatch with gpu sacrifices less wall-clock time to get a larger (step size) update.  However, in CNTK 2.2, I see that (among other function) `learning_rate_schedule` has been deprecated.

Should I be using the deprecated interface?  Is there a better way?

**Comment by tangyuq** [comment]
You can use [learning_parameter_schedule](https://cntk.ai/pythondocs/cntk.learners.html#cntk.learners.learning_parameter_schedule) to specify a learning rate schedule along with [minbiatch_size_schedule](https://cntk.ai/pythondocs/cntk.train.training_session.html?highlight=minibatch_size_schedule#cntk.train.training_session.minibatch_size_schedule)

**Comment by aschweig** [comment]
Thanks. I wasn't clear that the current period's minibatch_size from the schedule would flow into learning_parameter_schedule (or the learner for that matter) if set to None.  Seems to not be an issue.
